### PR TITLE
Listing flats

### PR DESCRIPTION
**Pull request by mcpenchel**
Scaffolded the flats (created model, controller and views), @obrenoco

### CODE REVIEW

File: app/controllers/flats_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  end
+
+  # POST /flats
+  # POST /flats.json
+  def create
+    @flat = Flat.new(flat_params)
+
+    respond_to do |format|
+      if @flat.save
+        format.html { redirect_to @flat, notice: 'Flat was successfully created.' }
```
Comment (mcpenchel) [inline app/controllers/flats_controller.rb]: Hey @mcpenchel , what is that? Do we really need it?

File: app/views/flats/_flat.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1,2 @@
+json.extract! flat, :id, :created_at, :updated_at
```
Comment (mcpenchel) [inline app/views/flats/_flat.json.jbuilder]: Man, _flat.json.jbuilder, wtf. I don't think we are gonna use this

File: app/views/flats/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (mcpenchel) [inline app/views/flats/_form.html.erb]: Your code is missing indentation

File: app/controllers/flats_controller.rb (hunk 1 above)
Comment (obrenoco) [reply app/controllers/flats_controller.rb]: OMG sorry matheus !! I'll fix it AASAP! @mcpenchel

**Comment by mcpenchel** [comment]
We are not using json, so remove that so I can merge it

**Pull request merged**